ansible/roles/sysctl/defaults/main.yml

```diff
@@ -235,6 +245,13 @@ sysctl__default_parameters:
                          ansible_virtualization_type == "openvz"))
                   | ternary("present", "absent") }}'
 
+      - name: 'net.ipv4.tcp_slow_start_after_idle'
+        value: '{{ sysctl__tcp_performance_enabled|bool | ternary(0, 1) }}'
```

Comment:
This `ternary()` function is redundant. If the option is not enabled, it will not be included in the config file. Just change this to `1` so that it's easier to parse by humans. Or should that be `0` to enable this behavior?

Reply:
The default value is 1. When set to 1, it slow starts. When set to 0 it does not slow start.

Reply:
If we play on the present/absent flag depending on the value of sysctl__tcp_performance_enabled, it means that if we deactivate the enabled flag, then the value is not reset to it's original value. This is also the problem with most other sysctl values.

eg:
sysctl__tcp_performance_enabled: true, => it sets net.ipv4.tcp_slow_start_after_idle to 0.
then
sysctl__tcp_performance_enabled: false, => it sets net.ipv4.tcp_slow_start_after_idle stays at 0.